### ISSUE TITLE
[Issue]: av1 and opus don't play direct

### ISSUE THREAD

**Issue by soorano** [jellyfin/jellyfin#12257]
### Please describe your bug

**_Expected:_** Play video direct without transcoding

**_What happend:_**
a video converted to av1 and opus don't play direct on TV and android. On Android there are 3 players you can select:
**Webplayer** plays the video but if it is 5.1 surround it won't downmix the audio and voices for example don't play.
The **integrated player** won't even start the video.
**External players** like VLC work fine.

If streaming is selected everything works.

### Reproduction Steps

1. Convert video to av1 and opus (I used FileFlows wich uses FFMPEG)
2. Put the video in directory from jellyfin
3. Try to play video on Android with different players (web, integrated, external) and on TV

### Jellyfin Version

10.9.0

### if other:

10.9.6

### Environment

```markdown
- OS: Debian 12, Docker install
- Linux Kernel:
- Virtualization: Docker
- Clients: Jellyfin TV (LG-Store), Android
- Browser:
- FFmpeg Version:
- Playback Method: Direct Play
- Hardware Acceleration: none
- GPU Model: none
- Plugins: none
- Reverse Proxy: traefik
- Base URL:
- Networking:
- Storage:
```

### Jellyfin logs

```shell
[2024-07-11 15:28:51.086 +00:00] [INF] [132] Emby.Server.Implementations.ScheduledTasks.TaskManager: "Datenbank optimieren" Completed after 0 minute(s) and 0 seconds
[2024-07-11 15:28:52.752 +00:00] [INF] [28] Emby.Server.Implementations.ScheduledTasks.TaskManager: "Aktualisiere Plugins" Completed after 0 minute(s) and 0 seconds
[2024-07-11 15:29:06.044 +00:00] [INF] [132] Jellyfin.Drawing.ImageProcessor: Creating image collage and saving to "/cache/temp/ffe5a6cb10dc478eafacbf973360969c.png"
[2024-07-11 15:29:06.928 +00:00] [INF] [132] Jellyfin.Drawing.ImageProcessor: Completed creation of image collage and saved to "/cache/temp/ffe5a6cb10dc478eafacbf973360969c.png"
[2024-07-11 15:29:06.946 +00:00] [INF] [28] Jellyfin.Drawing.ImageProcessor: Creating image collage and saving to "/cache/temp/0238869bd5d14238b7ac5d2c98deadf8.png"
[2024-07-11 15:29:07.783 +00:00] [INF] [28] Jellyfin.Drawing.ImageProcessor: Completed creation of image collage and saved to "/cache/temp/0238869bd5d14238b7ac5d2c98deadf8.png"
[2024-07-11 15:29:07.794 +00:00] [INF] [132] Jellyfin.Drawing.ImageProcessor: Creating image collage and saving to "/cache/temp/79a27497be5d436d91d3a338954be9f5.png"
[2024-07-11 15:29:08.413 +00:00] [INF] [132] Jellyfin.Drawing.ImageProcessor: Completed creation of image collage and saved to "/cache/temp/79a27497be5d436d91d3a338954be9f5.png"
[2024-07-11 15:29:08.425 +00:00] [INF] [132] Jellyfin.Drawing.ImageProcessor: Creating image collage and saving to "/cache/temp/369256a647e749debbb7f4972060007b.png"
[2024-07-11 15:29:09.297 +00:00] [INF] [132] Jellyfin.Drawing.ImageProcessor: Completed creation of image collage and saved to "/cache/temp/369256a647e749debbb7f4972060007b.png"
[2024-07-11 15:29:09.333 +00:00] [INF] [4] Emby.Server.Implementations.ScheduledTasks.TaskManager: "Scanne Medien-Bibliothek" Completed after 0 minute(s) and 18 seconds
[2024-07-11 15:29:09.426 +00:00] [INF] [25] Emby.Server.Implementations.IO.LibraryMonitor: Watching directory "/media/filme"
[2024-07-11 15:33:52.541 +00:00] [INF] [254] Emby.Server.Implementations.Session.SessionManager: Playback stopped reported by app "Jellyfin Android" "2.6.1" playing "Raya und der letzte Drache". Stopped at "2206836" ms
[2024-07-11 15:34:00.079 +00:00] [INF] [22] Jellyfin.Api.Helpers.MediaInfoHelper: User policy for "{USER}". EnablePlaybackRemuxing: True EnableVideoPlaybackTranscoding: True EnableAudioPlaybackTranscoding: True
```

### Please attach any screenshots here

External player on Android
![Screenshot_20240711-174157](https://github.com/jellyfin/jellyfin/assets/160236625/d757422f-21a6-4000-b226-51e1455175a2)

Webplayer audio don't play correct
![Screenshot_20240711-174126](https://github.com/jellyfin/jellyfin/assets/160236625/014465dd-8b3f-4c76-a03b-d7cb1cdd9300)
![Screenshot_20240711-170724](https://github.com/jellyfin/jellyfin/assets/160236625/50b29c12-24fd-4ea1-8e23-f84bfa9d9a48)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by soorano** [comment]
Hi, I don't know if you are sorting this issue to the right category. It is not an "android tv" problem. The clients where I tested the playback was an LG TV (OLED 65 CX) with "web os" installed from the LG store and to limit if the problem persists on other devices I tested it on an Android device, too.
I think it is more an issue on the **internal players** of the web os version and the android version because it works on external players but you are the experts...
